types/src/main.ts

```diff
@@ -2204,6 +2204,15 @@ export interface CodeAction {
 	 * executed and then the command.
 	 */
 	command?: Command;
+
+	/**
+     * Marks this as a preferred action. Preferred actions are used by the `auto fix` command and can be targeted
+     * by keybindings.
+     *
+     * A quick fix should be marked preferred if it properly addresses the underlying error.
+     * A refactoring should be marked preferred if it is the most reasonable choice of actions to take.
```

Comment:
Is it just me or does this indenting look off?

Reply:
It is a tab vs spaces problem. Apparently this files uses tabs. I'll fix it.